containers/base/build.sh

```diff
@@ -18,12 +18,17 @@
 # If [path_of_pydatalab_dir] is provided, it will copy the content of that dir into image.
 # Otherwise, it will get the pydatalab by "git clone" from pydatalab repo.
 
-# Build the docker image
+# Copy local pydatalab if supplied
 if [ -n "$1" ]; then
   rsync -avp "$1" pydatalab;
 else
   # Create empty dir to make docker build happy.
   mkdir -p pydatalab;
 fi
+
+# Copy supervisord files
+rsync -avp ../supervisor/ supervisor
```

Comment:
Adding more rsync calls to the build script makes me think the files are in the wrong place.

I wouldn't put any supervisor configs in the base image, but rather put the supervisor config for the Datalab frontend in the 'datalab' image and the other configs in the 'gateway' image.

That, in turn, means we would just have those files in the 'containers/datalab/content' and 'containers/gateway/content' directories.